pkgs/by-name/qs/qsv/package.nix

```
@@ -0,0 +1,158 @@
+{
+  fetchFromGitHub,
+  file,
+  pkg-config,
+  rustPlatform,
+  sqlite,
+  zstd,
+  lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "qsv";
+  version = "0.130.0";
+
+  src = fetchFromGitHub {
+    owner = "jqnatividad";
+    repo = "qsv";
+    rev = version;
+    hash = "sha256-GP3oMhYmOUsAYpn1XdvXFUw7imtOvajYApoLak4jpgc=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "calamine-0.25.0" = "sha256-LFQahQ+SfGWhp5Kfs7AJCj4zyxWiz099pwVuQucCF00=";
+      "dynfmt-0.1.5" = "sha256-/SrNOOQJW3XFZOPL/mzdOBppVCaJNNyGBuJumQGx6sA=";
+      "grex-1.4.5" = "sha256-3A1zn7anEPI6eSg5Lub9ZXJhTtVkvpRahM+b5GJjgHo=";
+      "local-encoding-0.2.0" = "sha256-ThXYKr3u/n2kvINcyobB2Ayex2sNbJEOyyjZH993Z4U=";
+      "polars-0.41.3" = "sha256-QD0hntNf0lQWQ02aPnYk8ZRQfrYnzIcZ8Kh2xObbWVY=";
+    };
+  };
+
+  buildInputs = [
+    file
+    sqlite
+    zstd
+  ];
```

Looks like we lost (or did we ever have them 😅) the Darwin frameworks?
```suggestion
  buildInputs =
    [
      file
      sqlite
      zstd
    ]
    ++ lib.optionals stdenv.isDarwin (
      with darwin.apple_sdk.frameworks;
      [
        AppKit
        CoreFoundation
        CoreGraphics
        IOKit
        Security
        SystemConfiguration
      ]
    );
```
Should be something like this, you'll need to add `stdenv` and `darwin` to the inputs though.

---

We did. I'll add this back.